microdnf/context.cpp

```diff
@@ -190,6 +191,13 @@ std::chrono::time_point<std::chrono::steady_clock> ProgressAndKeyImportRepoCB::p
 
 }  // namespace
 
+Context::Context() : plugins(std::make_unique<Plugins>(*this)) {}
+
+Context::~Context() {
+    // Objects from microdnf plugins must be destroyed before the plugins can be unloaded.
+    clear();
```

Comment:
This doesn't seem very nice. A reminder to discuss :slightly_smiling_face:

Reply:
Yes, from my point of view, it's terrible. I want to remake it later. I don't have a clear solution yet.

Reply:
Then maybe add a TODO and phrase the comment in such a way it's more apparent this isn't a good approach? (possibly along with a brief explanation or what's the issue)

Reply:
OK